py3status/modules/wwan.py

```diff
@@ -517,20 +504,41 @@ def wwan(self):
                     wwan_data['format_stats'] = self.py3.safe_format(
                         self.format_stats, stats)
 
+        # message and format message
+        if self.init['sms_message']:
+            message_data = self._get_message_data(modem_proxy)
+            wwan_data['message'] = len(message_data)
+            # format sms messages?
+            if self.init['format_message']:
+                wwan_data['format_message'] = self._manipulate_message(
+                    message_data)
+
+            # message urgent
+            last_count_message = self.py3.storage_get('count_message') or 0
+            count_message = wwan_data['message']
+            self.py3.storage_set('count_message', count_message)
+            if count_message > last_count_message:
+                urgent = True
+
         # thresholds
         for k, v in wwan_data.items():
             if isinstance(v, (float, int)):
                 self.py3.threshold_get_color(v, k)
 
-        # notify user
-        if self.init['message']:
-            if self.init['format_message']:
-                self._notify_message(wwan_data)
+        # ============================================
+        # EXPERIMENTING SOMETHING NEW. BREAKAGE AHEAD!
+        # ============================================
+        # notify users
+        if self.format_notification:
+            self.py3.notify_user(self.format_notification)
```

Comment:
```
self.py3.notify_user(self.py3.safe_format(self.format_notification))   # :-) 
```

Reply:
I don't need to pass data inside? I see no popup now

Reply:
Yes. Lol. Sorry I suck. Heading out for an hour. I will be back. Let me know if everything works... and what you're missing... and what you need. Leave comments in the code. I think for `{message}` (to get notifications). Maybe you could try an empty if... `[\?if=state=None {message}]` in there... so we can activate the `message`?  Idk. :man_shrugging: :woman_shrugging: